.github/workflows/node-ci.yml

```diff
@@ -48,15 +48,15 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
     - name: Install dependencies
       run: yarn install --frozen-lockfile
     - name: Run tests with coverage
       run: yarn coverage
     - name: Upload Coverage
-      uses: codecov/codecov-action@v1.0.5
```

Comment:
Being less specific should mean we'll use the latest version, which in this case, I think we want.